src/libstd/lib.rs

```diff
@@ -271,7 +271,6 @@
 #![feature(libc)]
 #![feature(link_args)]
 #![feature(linkage)]
-#![cfg_attr(not(stage0), feature(min_const_unsafe_fn))]
```

Comment:
Not sure if this is right; please double check.

Reply:
depending on when we get a new beta compiler, we'll need to just remove the `not`

Reply:
1.33 will go into beta roughly on 2019-03-01 -2 days == 2019-02-27;

Not sure how that impacts `stage0`.
Should I make it into `#![cfg_attr(stage0, feature(min_const_unsafe_fn))]` for now?

Reply:
No, that would fail to build. You need to do that if this PR goes to bors after the 27th